auth/src/main/java/org/jboss/pnc/auth/ExternalAuthFacade.java

```diff
@@ -52,7 +53,8 @@
      */
     public ExternalAuthFacade(InputStream keycloakConfiguration) throws Exception{
```

Comment:
It is already long there in the master and auth* is not actually part of this commit. We can open separate issue for this